Source/astcenc_vecmathlib.h

```diff
@@ -106,6 +114,33 @@
 	constexpr auto loada = vfloat4::loada;
 	constexpr auto load1 = vfloat4::load1;
 
+#elif ASTCENC_SVE == 8
+	/* If we have SVE configured as 8-wide, expose 8-wide VLA. */
+	#include "astcenc_vecmathlib_neon_4.h"
+	#include "astcenc_vecmathlib_common_4.h"
+	#include "astcenc_vecmathlib_sve_8.h"
+
+	/* Check the compiler is treating SVE as 256 bits ... */
+	#if __ARM_FEATURE_SVE_BITS != 256
```

Comment:
is there some possibility where this can be undefined by ASTCENC_SVE == 8?

perhaps better to have `#if !defined(__ARM_FEATURE_SVE_BITS) || (__ARM_FEATURE_SVE_BITS != 256)`

Reply:
Good shout. Adding that would catch cases where the user has set the ASTCENC_SVE but fudged the compile flags and not setting a fixed-length.